src/quo2/components/loaders/skeleton/component_spec.cljs

```diff
@@ -0,0 +1,13 @@
+(ns quo2.components.loaders.skeleton.component-spec
+  (:require [quo2.components.loaders.skeleton.view :as skeleton]
+            [quo2.foundations.colors :as colors]
+            [test-helpers.component :as h]))
+
+(h/describe "Skeleton tests"
+  (h/test "Skeleton component is animated when animated? is true"
```

Comment:
@FFFra, not related to your changes, but if we're setting a color on a quo2 component then most likely there is an issue here - these colors should be encapsulated by the design system in 99% of the cases. 
Could you take a look at what's there and make this set by some configuration rather than overrides?

Reply:
Yes! I will take a look and we can discuss it.

Reply:
I think it's because you are not testing outer component here actually 🤔

Reply:
Can you please explain me better? I was checking the code and I'm not sure if I understood. I'm sorry